recipes/GOFit/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "gofit" %}
+{% set version = "0.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/gofit-{{ version }}.tar.gz
+  sha256: ab5b578e641db5fe590627d0165abac67eaae9cc5a1451d3d6aceb3a2635e136
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+  host:
+    - python
+    - ninja
+    - pip
+    - pybind11
+    - eigen
+    - cmake
+    - make
+  run:
+    - python
+
+test:
+  source_files:
+    - tests
+  requires:
+    - pip
+    - pybind11
```

Comment:
Are `pybind11` and `eigen` needed to run the tests? If not, they should be removed.

Reply:
Thanks! They are not needed for the tests, @jess-farmer can we remove them from tests?